nulib/meadow
# Cutover Meadow to V2 indices

## &#x1F537; &#x1F537; TO DO ON THIS BRANCH BEFORE MERGING  &#x1F537; &#x1F537;

- [x] EDTF date csv export - blocked by: [#meadow/3887](https://github.com/nulib/repodev_planning_and_docs/issues/3887)
- [x] Thumbnails along top of batch edit screen
- [x] Homepage - visibility bar chart lost visibility labels on bottom axis
- [x] Bulk operations select by id's (Batch/csv export/shared links) sending old (v1 style) queries - select all bulk operations are OK
- [x] link into search to view works affected by batch and csv metadata updates, from their respective dashboards are using old v1 field name
- [x] Ingest Sheet and Ingest Project not appearing in facets
- [x] Link into search to view works associated with an ingest sheet is not working



# Summary 

Cuts over Meadow from the v1 (`meadow`) index to the new v2 indices: `dc-v2-work`, `dc-v2-collection` and `dc-v2-file-set`.

# Specific Changes in this PR

- Most updated the Search Results components, and the Homepage queries to pull repository stats.
- Backend configuration for v1 removed
- Batch update, csv export and other associated code updated to use new indices

# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test
Ensure home page stats seem accurate, and that search results load properly, faceting works, and so forth on the Search page.


# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes


# Tested/Verified
